src/test/emulators/auth/deleteAccount.spec.ts

```diff
@@ -103,4 +104,31 @@ describeAuthEmulator("accounts:delete", ({ authApi }) => {
         expect(res.body.error).to.have.property("message").equals("MISSING_LOCAL_ID");
       });
   });
+
+  it("should error if usageMode is passthrough", async () => {
+    const { idToken } = await registerUser(authApi(), {
+      email: "alice@example.com",
+      password: "notasecret",
+    });
+    await authApi()
+      .post("/identitytoolkit.googleapis.com/v1/accounts:delete")
```

Comment:
As discussed offline, this should error if deleting by idToken. In the case that the localId is used and passthrough is enabled, this call will still error but because the user will be missing. Added a test for the second case, but also leaving this test in